core/templates/pages/topics-and-skills-dashboard-page/skills-list/unassign-skill-from-topics-modal.controller.spec.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for UnassignSkillToTopicModalController.
+ */
+
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+describe('Unassign Skill from Topics Modal', function() {
+  var $scope = null;
+  var $rootScope = null;
+  var $q = null;
+  var $uibModalInstance = null;
+  var TopicsAndSkillsDashboardBackendApiService = null;
+  var topicIdDict = {
+    topic1: {id: 'tasd42'}
+  };
+
+  beforeEach(angular.mock.module('oppia'));
+
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
+    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
+      $provide.value(key, value);
+    }
+  }));
```

Comment:
Not sure if we are allowing this.. @nithusha21 to confirm.

Reply:
Fixed. Thanks!